src/App.vue

```diff
@@ -1,66 +1,59 @@
 <template>
   <v-app class="app-container" id="app">
     <!-- Dialogs -->
-    <file-and-pay-invalid-name-request-dialog
+    <FileAndPayInvalidNameRequestDialog
```

Comment:
I have decided that I prefer this name format as:
- it makes it easier to search where components are used
- VS Code highlights these in a contrasting colour, so they stand out against the other markup
- VS Code gives me contextual help (including Go To Reference)

Reply:
Are you guys OK with adopting this style for our components in the template?

Reply:
Totally :) I'm impartial!